zcash_client_backend/src/data_api/testing/pool.rs

```diff
@@ -238,3 +240,2302 @@ pub fn send_single_step_proposed_transfer<T: ShieldedPoolTester>(
         Ok(_)
     );
 }
+
+// #[cfg(feature = "transparent-inputs")]
+// pub(crate) fn send_multi_step_proposed_transfer<T: ShieldedPoolTester>() {
```

Comment:
Aha. Is what you did copy across all of the code, and comment it out at the destination in the same commit? Please do not do that, as the current approach makes it impossible to review the move.

Instead, either copy all the code as-is in one move (creating a commit that fails to compile) and then in subsequent commits apply the fixes, or move individual test functions over in one commit at a time.